x-pack/plugins/apm/public/components/routing/templates/apm_main_template.tsx

```diff
@@ -29,14 +32,34 @@ export function ApmMainTemplate({
   pageTitle?: React.ReactNode;
   pageHeader?: EuiPageHeaderProps;
   children: React.ReactNode;
-} & EuiPageTemplateProps) {
+} & KibanaPageTemplateProps) {
   const { services } = useKibana<ApmPluginStartDeps>();
 
   const ObservabilityPageTemplate =
     services.observability.navigation.PageTemplate;
 
+  // TODO: NEEDS A DATA GET A CHECK
+  const hasData = false;
+  const noDataConfig: KibanaPageTemplateProps['noDataConfig'] = {
```

Comment:
**APM**

@sqren Thanks for the info, but we need this new screen to be at the top level so that it takes over the **entire page**. This is the best location I found all of the APM screens. I just need help getting the logic check in here^^